lib/sles4sap/azure_cli.pm

```diff
@@ -215,6 +255,36 @@ sub az_network_publicip_create {
     assert_script_run($az_cmd);
 }
 
+
+=head2 az_network_publicip_get
+
+    az_network_publicip_get(
+        resource_group => 'openqa-rg',
+        name => 'openqa-pip')
+
+    Return an IPv4 public IP address from its name
```

Comment:
Is the indentation on purpose? It will cause the help message to be rendered as part of the code block IIRC.